projects/1.21/assets/the-undergarden/undergarden/lang/zh_cn.json

```
@@ -342,6 +417,9 @@
   "item.undergarden.cloggrum_axe": "扼塞斧",
   "item.undergarden.cloggrum_battleaxe": "扼塞战斧",
   "item.undergarden.cloggrum_boots": "扼塞靴子",
+  "item.undergarden.cloggrum_bucket": "扼塞桶",
+  "item.undergarden.cloggrum_bucket.block": "%s扼塞桶",
+  "item.undergarden.cloggrum_bucket.entity": "%s扼塞桶",

```

```suggestion
  "item.undergarden.cloggrum_bucket.block": "扼塞%s桶",
  "item.undergarden.cloggrum_bucket.entity": "扼塞%s桶",
```
我记得天镜的天根木桶是天根XX桶这种格式的